lib/cocoapods-playgrounds/workspace.rb

```diff
@@ -168,8 +174,14 @@ def generate_project
     def generate_swift_code(path)
       File.open(path + 'Contents.swift', 'w') do |f|
         f.write("//: Please build the scheme '#{target_name}' first\n")
-        f.write("import XCPlayground\n")
-        f.write("XCPlaygroundPage.currentPage.needsIndefiniteExecution = true\n\n")
+        if @use_legacy_xcode
```

Comment:
I'm currently initializing use_legacy_xcode based on the output of xcodebuild. Is there a better way to do this instead ?